c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -174,6 +174,8 @@ struct BlockPool {
   // Add a Block into blocks set with updating gc counter.
   std::pair<std::set<Block*, Comparison>::iterator, bool> insert_into_blocks(
       Block* block);
+
+  MempoolId_t owner_MempoolId() const;
```

Comment:
The weird casing of the method is "inspired" by the existing `owner_PrivatePool` field, since this method is basically just a shortcut for `->owner_PrivatePool->id`.